File: FootballApp/src/main/java/com/football/dev/footballapp/dto/InjuryDto.java
package com.football.dev.footballapp.dto;
import com.football.dev.footballapp.models.enums.InjuryStatus;
import lombok.*;

import java.time.LocalDate;
import java.util.Date;
@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
public class InjuryDto {
    private Long id;
    private String injuryType;
    private LocalDate injuryDate;
    private LocalDate expectedRecoveryTime;
    private InjuryStatus injuryStatus;
}

File: my-app/src/app/common/injury-dto.ts
import {InjuryStatus} from "../enums/injury-status";

export class InjuryDto {
    public id =0;
    constructor(public injuryType: string, public injuryDate: Date,public expectedRecoveryTime: Date,public injuryStatus: InjuryStatus) {
    }
}
